x/withdraw/types/params.go

```diff
@@ -0,0 +1,76 @@
+package types
+
+import (
+	"fmt"
+	"time"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// Parameter store key
+var (
+	ParamStoreKeyEnableWithdraw        = []byte("EnableWithdraw")
+	ParamStoreKeyPacketTimeoutDuration = []byte("PacketTimeoutDuration")
+)
+
+// ParamKeyTable returns the parameter key table.
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+// NewParams creates a new Params instance
+func NewParams(
+	enableWithdraw bool, timeoutDuration time.Duration,
+) Params {
+	return Params{
+		EnableWithdraw:        enableWithdraw,
+		PacketTimeoutDuration: timeoutDuration,
```

Comment:
Given interdependence on other modules, seems like `x/claims` will manage checking authorized channels in the IBC middleware for this module too? Is that correct?

Reply:
correct, it checks for authorized and EVM channels